### DISCUSSION TITLE
alarms and create_alarms functions

### DISCUSSION THREAD

**Discussion by Bruno21** [avantrec/soco-cli#45] [Ideas]
Hi there,

I've create an alarm with:

`07:00,01:30,WEEKDAYS,ON,"https://jazzradio.ice.infomaniak.ch/frequencejazz-high.mp3",NORMAL,50,OFF` as <alarm_spec>

When I display the alarm with the `list_alarms` function, i'got 'unknow' in the '5: Title' field. 
Is it possible to have the URI or the last part of URI instead ?

**Comment by pwt** [comment]
Hi Bruno:

Yes, it looks like that's possible. Would you prefer the whole URI to be displayed or just the final part?

**Reply by Bruno21** [reply]
in some cases the entire url can be very long, so just the last part is sufficient. Either way it will be perfect

**Reply by pwt** [reply]
Considering this a little more, I think one sometimes needs the whole URI for it to be helpful.
This what I've implemented in **v0.4.54**, which I've just released. Let me know if it works for you ...

**Reply by Bruno21** [reply]
It work very well ! The whole URI length is fine.
Many thanks :-)

**Reply by pwt** [reply]
Great -- thanks for testing.
